From 1003830e40ddbb01e907d00a10bbd1e3c6ab6ffa Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 23 Feb 2024 16:11:58 +0100
Subject: [PATCH 0936/1141] dt-bindings: display: update dt-bindings for STM32
 LVDS device

Add "st,stm32mp25-lvds" compatible for SOC MP25 & remove old one.

Change-Id: I84eb8e744895296739583c8de7313d8e67f06640
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/362683
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 Documentation/devicetree/bindings/display/st,stm32-lvds.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/display/st,stm32-lvds.yaml b/Documentation/devicetree/bindings/display/st,stm32-lvds.yaml
index 107841bedf91..5cf27a61bf86 100644
--- a/Documentation/devicetree/bindings/display/st,stm32-lvds.yaml
+++ b/Documentation/devicetree/bindings/display/st,stm32-lvds.yaml
@@ -20,7 +20,7 @@ properties:
     const: 0
 
   compatible:
-    const: st,stm32-lvds
+    const: st,stm32mp25-lvds
 
   reg:
     maxItems: 1
-- 
2.39.2

